datasets/au/dfat_sanctions/au_dfat_sanctions.yml

```
@@ -195,3 +195,39 @@ lookups:
         value: "1967-04-29"
       - match: 23-Apr-68
         value: "1968-04-23"
+      - match: 1970-1973
+        values: 
+          - 1970
+          - 1973
```

For the size of these ranges, it doesn't seem absurd to fill the gaps, does it?

```suggestion
          - 1970
          - 1971
          - 1972
          - 1973
```